build.gradle

```diff
@@ -37,11 +37,6 @@ subprojects { project ->
             !project.name.contains('examples')) {
         apply plugin: 'java-library'
 
-        // reactive-streams support will be published along with Spring Webflux 
-        if (!project.name.startsWith('rollbar-reactive-streams')) {
-            apply from: "$rootDir/gradle/release.gradle"
```

Comment:
I think we should keep this line but without the if statement

Reply:
Indeed... I'll fix this now.

Reply:
Fixed